dashboard/src/components/SearchFilter/SearchFilter.v2.tsx

```diff
@@ -0,0 +1,43 @@
+import { ClarityIcons, searchIcon } from "@clr/core/icon-shapes";
+import * as React from "react";
+import { CdsIcon } from "../Clarity/clarity";
+
+import "./SearchFilter.v2.css";
+
+ClarityIcons.addIcons(searchIcon);
+
+export interface ISearchFilterProps {
+  value: string;
+  className?: string;
+  placeholder: string;
+  onChange: (filter: string) => void;
+  onSubmit: (filter: string) => void;
+}
+
+function SearchFilter(props: ISearchFilterProps) {
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    props.onChange(e.currentTarget.value);
+  };
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    props.onSubmit(props.value);
+  };
+  return (
+    <div className="search-box">
+      <form onSubmit={handleSubmit}>
```

Comment:
Include the `aria-label` and `role` for accessibility:

```suggestion
      <form onSubmit={handleSubmit} role="search" aria-label="Search on the site">
```

With these changes, screen readers will notify the users this is a search box.